src/scanoss/file_filters.py

```diff
@@ -332,11 +364,43 @@ def get_filtered_files_from_files(self, files: List[str], operation_type: str, s
                     )
             except OSError as e:
                 self.print_debug(f'Error getting size for {rel_path}: {e}')
-
+        # End file loop
         return filtered_files
 
+    def _get_file_folder_pattern_spec(self, operation_type: str = 'scanning'):
+        """
+        Get file path pattern specification.
```

Comment:
we are using PEP 257 standard for docstrings as it seems to be the one used across new python scripts, as you can see in the other methods from this class and the new ones